crypto/bn/bn_recp.c

```diff
@@ -63,6 +63,7 @@ void BN_RECP_CTX_init(BN_RECP_CTX *recp)
     bn_init(&(recp->N));
     bn_init(&(recp->Nr));
     recp->num_bits = 0;
+    recp->shift = 0;
```

Comment:
yes, two PR's one for master and one for 1.0.2 is best, since the code is different.  line 66, above, as kurt says isn't needed in master.